Framework/NexusGeometry/src/NexusGeometryParser.cpp

```diff
@@ -543,6 +548,7 @@ parseNexusShape(const Group &detectorGroup) {
   }
   // Give shape group to correct shape parser
   if (shapeType == NX_CYLINDER) {
+    searchTubes = isGroup;
```

Comment:
yes could be tubes. not definite though.

Reply:
This is however the only scenario where one could have tubes in a Nexus file. The search which follows would then be definitive.